### PR TITLE
[INFOPLAT-1592] Address high CPU utilization when telemetry is enabled

### PR DESCRIPTION
**Pull request by pkcll**
**Context** 
 
High CPU usage for Telemetry reported by NOPs running Asset DON node.

Profiles examples taken from DON nodes:

image
image

It seems that NewBatchSpanProcessor is the offender

**Description**

- [loop/EnvConfig] parse sets TelemetryEmitterBatchProcessor, TelemetryEmitterExportTimeout
- [beholder/client] BatchProcessor ExportTimeout option is non-zero value

### CODE REVIEW

File: pkg/loop/config_test.go [hunk 1]
```diff
@@ -106,12 +137,66 @@ func TestEnvConfig_parse(t *testing.T) {
...
+	if len(a) != len(b) {
+		return false
+	}
+	for k, v := range a {
+		if b[k] != v {
+			return false
+		}
+	}
+	return true
+}
```
Comment (jmank88) [inline pkg/loop/config_test.go:198]: Does testify have helpers for comparing map entries?

File: pkg/beholder/client.go [hunk 2]
```diff
@@ -314,9 +321,12 @@ func newTracerProvider(config Config, resource *sdkresource.Resource, creds cred
 	if err != nil {
 		return nil, err
 	}
-
+	batcherOpts := []sdktrace.BatchSpanProcessorOption{}
+	if config.TraceBatchTimeout > 0 {
```
Comment (pkcll) [inline pkg/beholder/client.go:325]: The fix

File: pkg/loop/config_test.go (hunk 1 above)
Comment (cll-gg) [reply pkg/loop/config_test.go:198]: Why not use `reflect.DeepEqual()`?

Comment (pkcll) [reply pkg/loop/config_test.go]: Addressed https://github.com/smartcontractkit/chainlink-common/pull/967/commits/22280e07c68a8770b8e10ef4f0969553d63fd862

**Review by patrickhuie19 — APPROVED** [review]

**Review by 4of9 — APPROVED** [review]

**Pull request merged**